cluster/schema/manager.go

```diff
@@ -125,10 +125,16 @@ func (s *SchemaManager) PreApplyFilter(req *command.ApplyRequest) error {
 
 	// Discard adding class if the name already exists or a similar one exists
 	if req.Type == command.ApplyRequest_TYPE_ADD_CLASS {
-		if other := s.schema.ClassEqual(req.Class); other == req.Class {
-			return fmt.Errorf("class name %s already exists", req.Class)
+		other, is_alias := s.schema.ClassEqual(req.Class)
+		item := "class"
+		if is_alias {
```

Comment:
```suggestion
		other, isAlias := s.schema.ClassEqual(req.Class)
		item := "class"
		if isAlias {
```
nit: not to be that guy, but naming conventions